runner/tests/test_port.py

```diff
@@ -0,0 +1,43 @@
+import os
+from uuid import UUID
+from django.test import TestCase
+from django.conf import settings
+from django.core.management import call_command
+from file_system.models import File
+from runner.models import Port, PortType, Run
+
+class TestPorts(TestCase):
+    # load fixtures for the test case temp db
+    fixtures = [
+    "file_system.filegroup.json",
+    "file_system.filetype.json",
+    "file_system.storage.json",
+    "runner.pipeline.json"
+    ]
+
+    def test_true(self):
+        self.assertTrue(True)
+
+    def test_fixtures1(self):
+        """
```

Comment:
Same for this one?

Reply:
Trying to make sure that changes to the fixtures json files are recognized as part of the dependency for test. Since the fixtures are saved in external files to the tests, they could be be updated at some point in the future e.g. to reflect model schema changes. This test is trying to make some basic guarantees that the intended number of elements are still saved in the JSON file, otherwise it could affect downstream tests.

Reply:
Well if fixtures are updated in a way that test no longer works would the test fail? This test looks to me like you have a test to test would another test work. It is a regular thing that you need some kind of a mock data for tests so I think if anybody needs to fix the test which started failing, first thing that person would look at is, fixtures or setUp method. I prefer removing this as well.